### ISSUE TITLE
[1.11.2] The game crashes upon opening the GUI of the crusher

### ISSUE THREAD

**Issue by ghost** [rwtema/extrautilities_old_issues#1375]
This has been posted several times before (#1363, #1327, #1302, #1290, #1279, #1277), but I really want to bring attention to this issue, because it’s very annoying.

When the player crushes ores, exits the crusher’s GUI, and re-enters the GUI once the crushing is finished, Minecraft crashes. This also happens without OptiFine.

Log: http://pastebin.com/raw/0VcGxBfW

This only happens in Minecraft 1.11.2.

@RWTema, can you please fix this?

**Comment by ShelLuser** [comment]
I just tried but I can't reproduce this issue. That's not saying there's nothing wrong, but still. What exactly are you trying to crush? Also: what Forge version do you use (latest?).

EDIT: And I managed too. It happens once the crusher is finished.

EDIT2:

This is a duplicate of #1279.

Don't take my post in the wrong way please, I'm merely trying to help round things up a bit in order to make it easier for @rwtema to sort it all out. I can well imagine that it could become a little overwhelming when you have to sort out dozens of duplicates.

**Comment by ghost** [comment]
@ShelLuser It looks like you only read the title and not the actual issue; I _did_ mention that it only happens when the crusher is finished crushing, and I also acknowledged that this _is_ a duplicate of #1279, along with several other issues, but I posted it again because it _needs_ to be fixed; otherwise, it is essentially impossible for me to continue my modded 1.11.2 playthrough, and the same goes for many other people, as you can tell by all the duplicates of this issue.
